### PR TITLE
docs: rich text parser feature files

### PR DESCRIPTION
**Pull request by cooper-joe**
Feature files for the rich text parser. The features here could theoretically be included in ui-core, but planned features such as @mentions will include DHIS2 logic, making this a ui-widgets component.

For reference, the feature files are based on a custom implementation of [markdown-it](https://github.com/markdown-it/markdown-it).

@varl I noticed you already included some tests/specs in the d2ui rich text port [here](https://github.com/dhis2/ui-widgets/blob/port-rich-text/src/RichText/parser/__tests__/MdParser.spec.js). Many of the feature files I have written replicate these specs, so I think everything is covered. I'll leave it up to you whether to include both sets of features/specs.

### CODE REVIEW

**Comment by varl** [comment]
The "specs" there are unit tests, we will want them to guard the implementation.

**Comment by amcgee** [comment]
@cooper-joe is this something we should get in before deprecating this repository?  It looks like we'd need to mark these feature files as "not implemented yet" somehow so that they don't break the Cypress tests in the meantime (@Mohammer5 ?)

**Comment by HendrikThePendric** [comment]
@amcgee These feature files should end up in `cypress/drafts`

**Comment by cooper-joe** [comment]
I think we can implement this in `ui`, as long as we make sure to preserve all the specs/discussion here 👍

File: cypress/integration/RichTextParser/URL_Detection.feature [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Feature: Automatic url detection
+
+  Scenario Outline: Automatic URL detection
+      Given there is a text string that contains the URL of <URL>
+      And the URL is not preceded or followed immediately by a non-space character
+      Then the rendered URL is a <result>
+
+      Examples:
+        | URL                                        | result                                                                                              |
+        | duck.com                                   | <a href="http://duck.com">duck.com</a>                                                              |
```
Comment (amcgee) [inline cypress/integration/RichTextParser/URL_Detection.feature:10]: Complete tangent question - should we render these with `target="_blank"` with [`rel="noopener noreferrer"`](https://mathiasbynens.github.io/rel-noopener/)?  @cooper-joe

Comment (cooper-joe) [reply cypress/integration/RichTextParser/URL_Detection.feature]: Good question @amcgee, thanks for highlighting this. I think yes, those should be added by default. We can't know what user-generated content will be added here, so `_blank` is wise. Also definitely `rel="noopener noreferrer"`! I've added these to the scenario examples.

**Comment by HendrikThePendric** [comment]
Closing this one in favour of https://github.com/dhis2/ui/pull/130

**Pull request closed without merging**